storage/migrations/02_oasis_3_emerald.up.sql

```diff
@@ -111,12 +111,16 @@ CREATE INDEX ix_emerald_token_address ON oasis_3.emerald_token_balances (token_a
 CREATE TABLE oasis_3.emerald_tokens
 (
   token_address oasis_addr PRIMARY KEY,
+  discovery_state TEXT NOT NULL,
+  last_seen_round UINT63 NOT NULL,
```

Comment:
Oh now I see last week's update where it seems you intentinoally migrated this into a single table. Why did you prefer that? The state machine as you describe it can be implemented with a one table or two.

I think it's  cleaner in a separate table. Having shorter rows in the `emerald_tokens` table will also be helpful for faster joins, and we'll be joining this table to others a lot.
Ultimately, not a dealbreaker though.

Reply:
yeah two tables is fine. let's go with that

Reply:
went with separate table